docs/Introduction.md

```diff
@@ -1,21 +1,21 @@
 
 ## Key Features 
-- **Backed by Stripe**, #1 payment platform available in all BC SaaS countires.
-- Offer **flexible billing models** including tiered, per seat and usage based pricing.
-- Supports **135+ currencies** and multiple payment intervals.
-- No middle tier required. Only **direct**, **secure**, end-to-end **integration** with app owner’s Stripe account.
+- **Backed by Stripe**, the #1 payment platform available in all Business Central SaaS countries.
+- Offers **flexible billing models** including tiered, per seat and usage based pricing. Any model you choose to provide your customers.
+- Supports **135+ currencies** and multiple payment intervals. You select the currencies and payment intervals you accept.
+- Only **direct**, **secure**, **end-to-end** integration with the app publishers own Stripe account. No middle tier required.
 - A **seamless** user **sign-up** process onboards your customers within seconds.
-- **Secure customer's authentication** using single sign-on (SSO).
-- A **simple API** allows to integrate your extension with a few lines of code. 
+- **Secure customer authentication** using single sign-on (SSO).
+- A **simple API** allows you to integrate your extension with a few lines of code. 
 - A **single account** for all your apps **across any environment**.
 - Able to operate during network outages.
 
 ## Design Principles
-- **Security & Privacy**: Subscription Management takes a responsibility to connect app end-user with publisher. With this in mind we doing our best to guarantee this connection to be fully confidential and secure. We delivering this by making sure there are no midle-man in this connection and working closely with our [Security Team](https://www.theta.co.nz/solutions/cyber-security/) running penetration testing to limit risk of security vulnerabilities.
-- **Seamless Experience**: we think that seamless customer journey is a key to converting leads into a paid users. That's why we are conscious to deliver a best user experience across all application interfaces. That's related to app UI (follow conventions of design , minimalism and simplicity), API (simplified and well documented) as well overall onboarding experience. 
-- **Publisher's Identification**: Subscription Management is **publisher's tool** in a first place and should not present any other identity rather than publishers one. That's why we wiping ours from UI wherever possible.
+- **Security & Privacy**: Subscription Management takes on the responsibility for connecting Business Central end-users with app publishers. With this in mind we do our best to guarantee a fully secure and confidential connection. This is achieved by removing any midle-men from the connection in close collaboration with our [Security Team](https://www.theta.co.nz/solutions/cyber-security/). Part of our security hardening processes is e.g. penetration testing to reduce security vulnerabilities and risks.
```

Comment:
It's not about "reduce", but to eliminate any :)

Reply:
Sorry that is a promise you can never make in security. Ask Jeremy :)

Reply:
limit? Nice word for this.